src/components/buttons/ContinueButton.tsx

```diff
@@ -10,10 +9,10 @@ interface Props {
 // Splitting the CSS for the button link into two separate objects;
 // First, all the standard LinkCSS properties we need, so they can be typechecked...
 const anchorStyles: LinkCSS = {
-    color: "#ffffff",
     fontFamily: "'Guardian Text Sans',sans-serif",
     fontSize: "17px",
     lineHeight: "36px",
+    color: "#ffffff",
```

Comment:
TODO for us (but we can do in a different PR) to use palette colours here, rather than inlined hex values.

Reply:
Good spot, I've made the change.